src/components/PostInfo/PostInfo.scss

```diff
@@ -1 +1,16 @@
-//
```

Comment:
Use block comments instead of line comments to follow the given instructions. Replace '//' with '/* */'.